docs/vault/faq.md

```diff
@@ -32,13 +30,13 @@ We recommend Vaults remain online to avoid failing redeem requests in times of h
 
 ### What are collateral requirements?
 
-Vaults must lock up DOT collateral worth ``150%`` of the locked BTC value.
+Vaults must lock up collateral worth ``150%`` of the locked BTC value.
 
 ### What happens if a Vault fails to redeem BTC?
 
 The Vault will be slashed a punishment fee and the user can (i) chose to retry with another Vault or (ii) claim the Vault's collateral.
 
-### What happens if a vault steals?
+### What happens if a Vault steals?
 
 The Vault's collateral, up to ``150%`` of the stolen BTC value at the current exchange rate, is slashed and the interBTC bridge initiates a [**Burn Event**](/overview?id=burn-event-restoring-a-11-physical-peg).
```

Comment:
This also needs to be updated with the link to the thresholds since it depends on the collateral.